src/installation/guides/arm-devices/index.md

```diff
@@ -0,0 +1,137 @@
+utility, for example.
+
+To access the newly created file systems, it is necessary to mount them. This
+guide will assume that the second partition will be mounted on `/mnt`, but you
+may mount it elsewhere. To mount these filesystems, you can use the commands
+below, replacing the device names with the appropriate ones for your setup.
+
+```
+# mnt /dev/mmcblk0p2 /mnt
+# mkdir /mnt/boot
+# mount /dev/mmcblk0p1 /mnt/boot
+```
+
+### Tarball installation
+
+First, [download and verify](../../index.md#downloading-installation-media) a
+PLATFORMFS or ROOTFS tarball for your desired platform and [prepare your storage
+medium](#preparing-your-storage). Then, it is necessary to unpack the tarball
+onto the file system, which can be done using
+[tar(1)](https://man.voidlinux.org/tar.1):
+
+```
+# tar xvfp <image>.tar.xz -C /mnt
+```
+
+### Chroot installation
+
+It is also possible to perform a chroot installation, which requires using the
+`qemu-user-static` package together with either the `binfmt-support` or `proot`
+package. This guide explains how to use the `qemu-<platform>-static` program
+from `qemu-user-static` with [proot(1)](https://man.voidlinux.org/proot.1).
+
+First, [download and verify](../../index.md#downloading-installation-media) a
+PLATFORMFS or ROOTFS tarball for your desired platform and [prepare your storage
+medium](#preparing-your-storage). Then, follow the [XBPS chroot installation
+steps](../chroot.md#the-xbps-method) using the appropriate architecture and base
+packages, some of which are listed in the "[Supported
+platforms](./platforms.md)" page.
+
+Finally, follow the [chroot configuration steps](../chroot.md#configuration)
+steps, but instead of using the [chroot(1)](https://man.voidlinux.org/chroot.1)
+command to [enter the chroot](../chroot.md#entering-the-chroot), use the command
+below:
+
+```
+proot -q qemu-arm-static -r /mnt -w /
+```
+
+## Configuration
+
+Some additional configuration steps need to be followed to guarantee a working
+system. Configuring a [graphical
+session](../../../config/graphical-session/index.md) should work as normal.
+
+### Logging in
+
+The `root` password is `voidlinux`.
+
+### fstab
+
+The `/boot` partition must be added to `/etc/fstab`, with an entry similar to
+the one below. If you aren't using an SD card, replace `/dev/mmcblk0p1` with the
+appropriate device.
+
+```
+/dev/mmcblk0p1 /boot vfat defaults 0 0
+```
+
+### System time
+
+Several of the ARM devices supported by Void Linux don't have battery powered
+real time clocks (RTC), which means they won't keep track of time once powered
+off. This issue can present itself as HTTPS errors when browsing the web or
+using the package manager. It is possible to set the time manually using the
+[date(1)](https://man.voidlinux.org/date.1) utility. In order to fix this issue
+for subsequent boots, install and enable [an NTP
+server](../../../config/date-time.md#ntp). Furthermore, it is possible to
```

Comment:
Nitpick: "client" or "daemon" would be a better fit.

Reply:
Thanks!